Source/Core/DolphinQt2/Settings/WiiPane.cpp

```diff
@@ -0,0 +1,77 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "DolphinQt2/Settings/WiiPane.h"
+
+#include <QCheckBox>
+#include <QGroupBox>
+#include <QLabel>
+#include <QSlider>
+#include <QVBoxLayout>
+#include <QWidget>
+
+#include "Core/ConfigManager.h"
+
+WiiPane::WiiPane(QWidget* parent) :QWidget(parent)
+{
+  CreateLayout();
+  ConnectLayout();
+
+  LoadConfig();
+}
+
+void WiiPane::CreateLayout()
+{
+  m_main_layout = new QVBoxLayout;
+  CreateMisc();
+  m_main_layout->setContentsMargins(0, 0, 0, 0);
+  m_main_layout->addStretch(1);
+  setLayout(m_main_layout);
+}
+
+void WiiPane::ConnectLayout()
+{
+  connect(m_wiimote_speaker_volume, &QSlider::valueChanged, this, &WiiPane::OnSaveConfig);
+  connect(m_wiimote_motor, &QCheckBox::toggled, this, &WiiPane::OnSaveConfig);
+}
+
+void WiiPane::CreateMisc()
+{
+  auto* basic_group = new QGroupBox(tr("Misc Settings"));
+  auto* basic_group_layout = new QVBoxLayout;
+  basic_group->setLayout(basic_group_layout);
+  m_main_layout->addWidget(basic_group);
+  m_wiimote_motor = new QCheckBox(tr("Enable Wiimote Motor"));
+  // Speaker Volume Slider
+  m_wiimote_speaker_volume_label = new QLabel(tr("Wiimote Speaker Volume"));
+  m_wiimote_speaker_volume = new QSlider(Qt::Horizontal);
+  m_wiimote_speaker_volume->setFocusPolicy(Qt::StrongFocus);
+  m_wiimote_speaker_volume->setTickPosition(QSlider::TicksBothSides);
+  m_wiimote_speaker_volume->setTickInterval(8);
+  m_wiimote_speaker_volume->setSingleStep(1);
+  m_wiimote_speaker_volume->setMinimum(0);
+  m_wiimote_speaker_volume->setMaximum(127);
+
+  basic_group_layout->addWidget(m_wiimote_speaker_volume_label);
+  basic_group_layout->addWidget(m_wiimote_speaker_volume,1, Qt::AlignRight);
```

Comment:
Using a QVBoxLayout doesn't quite accomplish what we want here: even though the slider is being aligned right, it gets shunted onto a separate line.

A pattern used elsewhere for this type of UI is to use a QGridLayout:

```c++
auto* basic_group_layout = new QGridLayout();
basic_group_layout->addWidget(m_wiimote_speaker_volume_label, 0, 0);
basic_group_layout->addWidget(m_wiimote_speaker_volume, 0, 1, 1, -1);
basic_group_layout->addWidget(m_wiimote_motor, 1, 0, 1, -1);
```

You could also make it work by adding the widgets on that row into a QHBoxLayout, and then adding that into the basic group layout.